server/sandbox_run.go

```diff
@@ -69,18 +69,23 @@ func (s *Server) RunPodSandbox(ctx context.Context, req *types.RunPodSandboxRequ
 	return s.runPodSandbox(ctx, req)
 }
 
-func convertPortMappings(in []*types.PortMapping) []*hostport.PortMapping {
-	out := make([]*hostport.PortMapping, 0, len(in))
-	for _, v := range in {
-		if v.HostPort <= 0 {
-			continue
+func convertPortMappings(in []*types.PortMapping, disableHostPortMapping bool) []*hostport.PortMapping {
+	var out []*hostport.PortMapping
+	if disableHostPortMapping {
+		out = []*hostport.PortMapping{}
```

Comment:
typically, I prefer to return early rather than store a variable and have if/elses. Can this be:
```
if disableHostPortMapping {
    return []*hostport.PortMapping{}
}
...
```

Reply:
Done